stays/views.py

```diff
@@ -1,3 +1,60 @@
-from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
+from django.db.models import Min, Max, Prefetch
 
-# Create your views here.
+from .models import Stay, StayImage, Room
+
+class StayDetailView(View):
+    def get(self, request, stay_id):
+        try:
+            stay = Stay.objects.select_related(
+                'stay_type'
+                ).annotate(
+                minprice    = Min('room__roomoption__price'),
+                maxprice    = Max('room__roomoption__price'),
+                mincapacity = Min('room__min_capacity'),
+                maxcapacity = Max('room__max_capacity'), 
+                ).prefetch_related(
+                    Prefetch('stayimage_set', StayImage.objects.all(), to_attr='stayimages'),
+                    Prefetch('room_set', Room.objects.all(), to_attr='rooms')
+                ).get(id = stay_id
+                )
+                
+            result  = [{
+                'stay_id'      : stay.id,
+                'stay_name'    : stay.name,
+                'stay_image'   : [image.image for image in stay.stayimages],
```

Comment:
```suggestion
                'stay_image'   : [{
                    'id' : image.id,
                    'url' : image.image
                    }for image in stay.stayimages],
```
`image, room, feature, option`과 같이 고유한 객체들은 id(pk)값을 같이 반환해주는 것이 일반적입니다!

Reply:
감사합니다. 모두 반영하겠습니다.